gamestonk_terminal/cryptocurrency/README.md

```diff
@@ -6,9 +6,21 @@ This menu aims to explore crypto world, and the usage of the following commands
   * load a given coin vs a given currency [CoinGecko]
 * [view](#view)
   * plot the loaded crypto data
+* [trend](#trend)
```

Comment:
Could we here distinguish between data providers?

Similarly to Behaviour Analysis README